base_warn_option/readme/CONTEXT.md

```diff
@@ -0,0 +1,12 @@
+This module has been developed to maintain consistency in the messages used when
+blocking some models.
+
+This module doesn't do anything by itself.
```

Comment:
At least, it adds some models, views and permissions. You could include configuration instructions, so you don't need to repeat them on each of the upcoming submodules.

Reply:
Permissions are not necessary because this field will go into each <group> on the models and the <group> has the related permission itself.

Views are created but are not used (only form when opening the record on a pop-up).

There is nothing to configure.